From 3dee3e3890c87338a7beeb956bd2a37ecd8fda63 Mon Sep 17 00:00:00 2001
From: Keith Mok <kmok@cyngn.com>
Date: Fri, 23 Sep 2016 16:32:48 -0700
Subject: [PATCH 041/249] Add google/contexthub

Some device depends on it. (e.g. bullhead)
Otherwise sensor will not work.

Change-Id: I56fa4a8ce3f44f035011714c4085150de60b3c37
---
 default.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/default.xml b/default.xml
index fce5dfc..9b13289 100644
--- a/default.xml
+++ b/default.xml
@@ -49,6 +49,7 @@
   <project path="device/generic/qemu" name="device/generic/qemu" remote="aosp" />
   <project path="device/generic/x86" name="device/generic/x86" groups="pdk" remote="aosp" />
   <project path="device/generic/x86_64" name="device/generic/x86_64" groups="pdk" remote="aosp" />
+  <project path="device/google/contexthub" name="device/google/contexthub" groups="device" remote="aosp" />
   <project path="device/sample" name="device/sample" groups="pdk" remote="aosp" />
   <project path="external/aac" name="CyanogenMod/android_external_aac" groups="pdk" />
   <project path="external/adt-infra" name="platform/external/adt-infra" groups="adt-infra,notdefault" remote="aosp" />
-- 
2.17.1

